src/Persistence/Models/Component.cs

```diff
@@ -24,4 +24,7 @@ public Component(string name, string variant, IControlTemplateStore controlTempl
         Variant = variant;
         Template = controlTemplateStore.GetByName(BuiltInTemplates.Component);
     }
+
+    [YamlMember(Order = 4)]
```

Comment:
It might be useful to set it to 100 to create room for more properties in base class